### PR TITLE
Minor update to finetuning scripts

### PR DESCRIPTION
**Pull request by anwai98**

### CODE REVIEW

**Comment by anwai98** [comment]
GTG from my side!

File: finetuning/.gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@ sam_embeddings/
 results/
 iterative_prompting_results/
 *.sh
+run_all_finetuning.py
```
Comment (constantinpape) [inline finetuning/.gitignore:7]: Why do we add this to the gitignore? If it's a script shouldn't we just add it?

**Review by constantinpape — COMMENTED** [review]
Looks good overall. Just a minor question why we exclude one of the scripts.

File: finetuning/.gitignore (hunk 1 above)
Comment (anwai98) [reply finetuning/.gitignore]: I wasn't sure if I should add it. It's a script to run all the finetunings at once. I added it just now!

**Pull request merged**